lib/assets/blog-article.mst

```diff
@@ -5,9 +5,9 @@
 </head>
 
 <body>
-<header class="extra-header">
```

Comment:
Why the renames?

In any case, you'll need to update the `<script>` elements in these files since they use the IDs.